utils/metrics.go

```diff
@@ -892,6 +906,8 @@ var metricsDefs = map[MetricName]metricDefinition{
 	},
 }
 
+var brokerMetricsDefs = map[MetricName]metricDefinition{}
```

Comment:
empty?

Reply:
yeah for now.
will add actual metrics when needed.